models/polls.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  name: "polls",
+  schema: {
+    name: String,
+    creator: String,
+    createdBy: {
+      type: String,
+      required: true,
+      index: true
+    },
+    questions: Array,
+    link: {
+      type: String,
+      index: true
+    },
+    result: Object,
+    numberOfResponses: Number,
+    metaInformation: Object,
```

Comment:
@deepakr123 - Please add the logic of putting all the form fields which are not there in the polls model in the metaInformation

Reply:
resolved